src/Compilers/Core/Portable/Symbols/Attributes/CommonTypeEarlyWellKnownAttributeData.cs

```diff
@@ -109,5 +109,26 @@ public bool HasCodeAnalysisEmbeddedAttribute
             }
         }
         #endregion
+
+        #region AsyncMethodBuilderAttribute
+
+        private TypedConstant _asyncMethodBuilderTarget = default;
```

Comment:
>private TypedConstant _asyncMethodBuilderTarget = default; [](start = 8, length = 58)

Do we have similar problem in VB? If we don't need this field for VB, we should move it to C# specific type.